src/main/java/heat/dto/Gender.java

```diff
@@ -0,0 +1,17 @@
+package heat.dto;
+
+public enum Gender {
```

Comment:
Let's start from 0:
FEMALE(0),
MALE(1),
OTHER(2);

Reply:
I suggest to use 0 as `OTHER` as 0 is usually used as the default value.

Reply:
Done.